governance.md

```diff
@@ -0,0 +1,393 @@
+Council would be consulted on this decision. If the BDFL is unable to appoint a
+successor, the Steering Council will make a suggestion or suggestions to the
+Main NumFOCUS Board. While the Steering Council and Main NumFOCUS Board will
+work together closely on the BDFL selection process, the Main NumFOCUS Board
+will make the final decision.
+
+
+Steering Council
+----------------
+
+The Project will have a Steering Council that consists of Project Contributors
+who have produced contributions that are substantial in quality and quantity,
+and sustained over at least one year. The Steering Council consists of the
+owners of the project organization. The overall role of the Council is to
+ensure, through working with the BDFL and taking input from the Community, the
+long-term well-being of the project, both technically and as a community.
+
+During the everyday project activities, council members participate in all
+discussions, code review and other project activities as peers with all other
+Contributors and the Community. In these everyday activities, Council Members
+do not have any special power or privilege through their membership on the
+Council. However, it is expected that because of the quality and quantity of
+their contributions and their expert knowledge of the Project Software and
+Services that Council Members will provide useful guidance, both technical and
+in terms of project direction, to potentially less experienced contributors.
+
+The Steering Council and its Members play a special role in certain situations.
+In particular, the Council may:
+
+-   Make decisions about the overall scope, vision and direction of the
+    project.
+-   Make decisions about strategic collaborations with other organizations or
+    individuals.
+-   Make decisions about specific technical issues, features, bugs and pull
+    requests. They are the primary mechanism of guiding the code review process
+    and merging pull requests.
+-   Make decisions about the Services that are run by The Project and manage
+    those Services for the benefit of the Project and Community.
+-   Make decisions when regular community discussion doesn’t produce consensus
+    on an issue in a reasonable time frame.
+
+### Council membership
+
+To become eligible for being a Steering Council Member an individual should
+meet some of the following criteria:
+
+-  Be a Project Contributor who has produced contributions that are substantial in
+quality and quantity
+-  Sustain this development consistently over several months
+-  Support the community through various activities (some examples below):
+   - code review
+   - answering user questions
+   - triaging bug reports
+   - participating constructively in broader conversations
+   - contributing to documentation
+   - maintaining infrastructure
+-  Demonstrate breadth by supporting the community outside of their particular
+   work interests or sub-project
+-  Be civil in public discourse
```

Comment:
Not have a conflict of interest?

Reply:
Do you have any potential conflicts of interest with being a SC member in mind? I'm not sure what one would look like.

Reply:
Given the section below, it seems like having a conflict of interest isn't a problem as long as it's disclosed.

Reply:
Oh!  Then this could be "Maintains open disclosure of conflicts of interest."  This brings up one more comment below.

Reply:
That probably makes sense for new council members. Though I wouldn't really want to require it of arbitrary people contributing to the project.